.github/workflows/.zipignore

```diff
@@ -0,0 +1,4 @@
+.git/*
+.github/*
```

Comment:
Impact of this one is negligible, and it may cause problems - e.g. when we create a package, and then have to use some internal actions down the pipeline.